applications/nrf_desktop/description.rst

```diff
@@ -0,0 +1,1082 @@
+   This report is handled using the configuration channel's infrastructure and you can enable it using the :ref:`CONFIG_DESKTOP_CONFIG_CHANNEL_OUT_REPORT <config_desktop_app_options>` Kconfig option.
+   See the Kconfig option's help for details about the report.
+
+HID protocols
+-------------
+
+The following HID protocols are supported by nRF Desktop for HID input and output reports:
+
+* Report protocol - Most widely used in HID devices.
+  When establishing a connection, the host reads a HID descriptor from the HID device.
+  The HID descriptor describes the format of HID reports and is used by the host to interpret the data exchanged between the HID device and the host.
+* Boot protocol - Only available for mice and keyboards data.
+  No HID descriptor is used for this HID protocol.
+  Instead, fixed data packet formats must be used to send data between the HID device and the host.
+
+.. _nrf_desktop_requirements:
+
+Requirements
+************
+
+The nRF Desktop application supports several development kits related to the following hardware reference designs.
+Depending on the development kit you use, you need to select the respective configuration file and :ref:`build type <nrf_desktop_requirements_build_types>`.
+
+.. tabs::
+
+   .. tab:: Gaming mouse
+
+      .. figure:: /images/nrf_desktop_gaming_mouse_top_no_captions.svg
+         :alt: nRF Desktop gaming mouse (top view)
+
+      .. table-from-rows:: /includes/sample_board_rows.txt
+         :header: heading
+         :rows: nrf52840gmouse_nrf52840
+
+   .. tab:: Desktop mouse
+
+      .. figure:: /images/nrf_desktop_desktop_mouse_side_no_captions.svg
+         :alt: nRF Desktop desktop mouse (side view)
+
+      .. table-from-rows:: /includes/sample_board_rows.txt
+         :header: heading
+         :rows: nrf52dmouse_nrf52832, nrf52810dmouse_nrf52810
+
+   .. tab:: Keyboard
+
+      .. figure:: /images/nrf_desktop_keyboard_top_no_captions.svg
+         :alt: nRF Desktop keyboard (top view)
+
+      .. table-from-rows:: /includes/sample_board_rows.txt
+         :header: heading
+         :rows: nrf52kbd_nrf52832
+
+   .. tab:: HID dongle
+
+      .. figure:: /images/nrf_desktop_dongle_no_captions.svg
+         :alt: nRF Desktop dongle (top view)
+
+      .. table-from-rows:: /includes/sample_board_rows.txt
+         :header: heading
+         :rows: nrf52840dongle_nrf52840, nrf52833dongle_nrf52833, nrf52820dongle_nrf52820
+
+   .. tab:: DK
+
+      .. figure:: /images/nrf_desktop_nrf52840_dk_no_captions.svg
+         :alt: DK
+
+      .. table-from-rows:: /includes/sample_board_rows.txt
+         :header: heading
+         :rows: nrf52840dk_nrf52840, nrf52833dk_nrf52833, nrf52833dk_nrf52820, nrf5340dk_nrf5340_cpuapp
+
+      Depending on the configuration, a DK may act either as mouse, keyboard or dongle.
+      For information about supported configurations for each board, see the :ref:`nrf_desktop_board_configuration_files` section.
+
+..
```

Comment:
I will keep this as is for now. Not sure if this will break documentation rendering. I will check this in detail and if it is not needed, will fix this in a separate PR.